pyspeckit/spectrum/models/h2co_mm.py

```diff
@@ -102,18 +102,15 @@ def h2co_mm_radex(xarr,
 
     # Convert X-units to frequency in GHz
     xarr = xarr.as_unit('Hz', quiet=True)
-    Tex303,Tex322,Tex321,tau303,tau322,tau321 = gridbundle
 
-
-    # if this gets too far different from 1, we are gonna have a Bad Time.
-    scalefac = grid_vwidth/width
-
-    tex = (Tex303(logColumn,logDensity,Temperature),
-           Tex322(logColumn,logDensity,Temperature),
-           Tex321(logColumn,logDensity,Temperature))
-    tau =  (tau303(logColumn,logDensity,Temperature)*scalefac,
-            tau322(logColumn,logDensity,Temperature)*scalefac,
-            tau321(logColumn,logDensity,Temperature)*scalefac)
+    ModelDict = gridbundle(Temperature = Temperature,
+                           logColumn = logColumn,
+                           logDensity = logDensity,
+                           logwidth = np.log10(width))
+    
+    tex = (ModelDict['Tex303'],ModelDict['Tex322'],ModelDict['Tex321'])
+    tau = (ModelDict['tau303'],ModelDict['tau322'],ModelDict['tau321'])
```

Comment:
Why's the scale factor gone?  I think that's still needed.  Otherwise, `gridvwidth` should be ditched too